spec/acceptance/listen_spec.rb

```diff
@@ -7,6 +7,15 @@
   let(:all_options) { base_options.merge(polling_options).merge(options) }
 
   let(:wrapper) { setup_listener(all_options, :track_changes) }
+  around do |example|
+    warn_level = $VERBOSE
+    begin
+      $VERBOSE = nil
+      example.run
+    ensure
+      $VERBOSE = warn_level
+    end
+  end
```

Comment:
What kinds of warnings? I don't see any in the Travis build log (from the earlier commit of yours).

If you really need this, I think it's best to mark which specs need this work around and use RSpec metadata to set `$VERBOSE` only on those tests. (see: https://www.relishapp.com/rspec/rspec-core/docs/metadata/user-defined-metadata).

You may also only set `$VERBOSE` for OSX at those versions.

I don't mind it being ugly - just it's not useful to make it global and on every OS the tests are run. Especially since I don't know which warnings and which specs need this.